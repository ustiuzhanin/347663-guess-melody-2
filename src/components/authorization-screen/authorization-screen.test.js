import React from "react";
import renderer from "react-test-renderer";
import AuthorizationScreen from "./authorization-screen.jsx";

test(`AuthorizationScreen's snapshot`, () => {
  const tree = renderer.create(
      <AuthorizationScreen onChange={jest.fn()} onSubmit={jest.fn()} />
  );

  expect(tree).toMatchSnapshot();
});
