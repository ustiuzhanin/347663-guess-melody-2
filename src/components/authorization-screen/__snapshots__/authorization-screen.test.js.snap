// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorizationScreen's snapshot 1`] = `
<label
  className="login"
>
  <div
    className="login__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="login__title"
  >
    Fake authorization required!
  </h2>
  <p
    className="login__text"
  >
    Please introduce yourself!
  </p>
  <form
    action=""
    className="login__form"
    onChange={[MockFunction]}
    onSubmit={[MockFunction]}
  >
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="name"
      >
        Email
      </label>
      <input
        className="login__input"
        id="name"
        name="email"
        type="text"
      />
    </p>
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="password"
      >
        Password
      </label>
      <input
        className="login__input"
        id="password"
        name="password"
        type="text"
      />
      <span
        className="login__error"
      >
        Wrong Password
      </span>
    </p>
    <button
      className="login__button button"
      type="submit"
    >
      Sign In
    </button>
  </form>
</label>
`;
